secure_message/repository/modifier.py

```diff
@@ -91,12 +91,14 @@ def del_draft(draft_id):
         del_draft_status = "DELETE FROM securemessage.status WHERE msg_id='{0}' AND label='{1}'".format(draft_id, Labels.DRAFT.value)
         del_draft_event = "DELETE FROM securemessage.events WHERE msg_id='{0}'".format(draft_id)
         del_draft_inbox_status = "DELETE FROM securemessage.status WHERE msg_id='{0}' AND label='{1}'".format(draft_id, Labels.DRAFT_INBOX.value)
+        del_actors = "DELETE FROM securemessage.actors where msg_id='{0}'".format(draft_id)
         del_draft_msg = "DELETE FROM securemessage.secure_message WHERE msg_id='{0}'".format(draft_id)
 
         try:
             db.get_engine(app=db.get_app()).execute(del_draft_status)
```

Comment:
Isn't this where we should be using db.session?

Reply:
Possibly , I must admit I followed what was there. Not sure of the implications of changing to that . The draft is normally deleted in the context of sending the same message as a draft .Other functions in the same class do use the session so Its possible that the two might interfere with each other , but unless I did an investigation I could not be sure .

Reply:
Session should be used. It encapsulates db.get_engine. See https://github.com/mitsuhiko/flask-sqlalchemy/issues/107 for it's implementation.

Reply:
Added as a tech debt issue